examples/BlazorServerApp/BrowserFilter.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.FeatureManagement;
```

Comment:
Do we need to include this custom filter in this example? It's already available in the FeatureFlag demo.

Reply:
Yes. So previously, we need to call AddFeatureFilter for built-in filters. But now, only custom feature filters will be registered through our AddFeatureFilter API. The thing is that I want to give the information that, once the feature management is registered as scoped, all feature filters, targetingContextAccessor will be registered as scoped as long as they are registered through our APIs (i.e. AddFeatureFilter, WithTargeting)

Reply:
Also, the BrowerFilter shows a way how people can build a feature filter with HttpContext. You will need the user-agent information from the http context.